scripts/aggregate_slurm.sh

```diff
@@ -0,0 +1,134 @@
+echo "Done study cleanup"
+
+# Generate a string listing directories with results to be aggregated. 
+# Also copy irap.versions.tsv files to $workingDir - all of them should be
+# the same for each library of the same study. If not, copy each different
+# version of this file into irap.versions.tsv.<version> - for inspection
+# later - and log there's more than one version of the file
+
+workingDir=$(process_working_dir "$studyId" aggregate "$organism" 'yes')
+versions="${workingDir}/irap.versions.tsv"
+versionString=
+
+libraryPathsForStudy=
+
+# Try to get libraries from SDRF files first (preferred)
+
+libs=$(get_libraries_for_ae2_experiment $studyId)
+
+if [ $? -ne 0 ]; then
+
+    # Getting libs from the config SDRF failed, so fall back to using the 
+    # config XML file to get runs. This happens when we get crufty SDRF files,
+    # which is sometimes be unavoidable for controlled-access experiments,
+    # e.g., the expt is a mix of public (ENA) and controlled-access libraries.
+    warn "Failed to derive runs from SDRF file for $studyId, fetching all libraries from the config XML file instead."
+    libs=$(grep -P "<assay[^>]{0,}>[^<]+</assay>" $ATLAS_PROD/analysis/baseline/rna-seq/experiments/$studyId/$studyId-configuration.xml | awk -F '[<>]' '{ print $3 }')
+    
+    if [ $? -ne 0 ]; then
+        warn "[ERROR] Can't get libraries to aggregate for ${studyId}-${organism}"
+        return 1
+    fi
+fi
+
+while read -r library; do
+    resultsDir_ini=$(process_results_dir $library irap_single_lib $organism no no)
+    resultsDir=$(echo "$resultsDir_ini" | sed 's/\/[0-9]\{3\}\//\//')
+    libraryPathsForStudy="$libraryPathsForStudy $resultsDir"
+
+    # Check versions. Take out the parameters, which aren't helpful for
+    # users anyway, specifying config file paths etc. Deal with either
+    # versions.tsv or irap.vesions.tsv, whichever is available
+ 
+    versionsFile="${resultsDir}/versions.tsv"
+    if [ ! -s "$versionsFile" ]; then
+        versionsFile="${resultsDir}/irap.versions.tsv"
+    fi
+
+    libVersions=$(cat "$versionsFile" | grep -v "iRAP params")
+    
+    if [ -z "$versionString" ]; then
+        versionString=$libVersions
+        echo -e "$versionString" >> $versions
+    elif [ "$libVersions" != "$versionString" ]; then
+        echo -e "$versionString" >> ${versions}.${library}
+    fi
+
+done <<< "$libs"
+
+# Write library paths to file to pass to iRAP for aggregation
+
+aux=${ISL_TEMP_DIR}/aux.${studyId}.${organism}.$$
+echo $libraryPathsForStudy > $aux
+            
+# Submit aggregation to cluster
+
+aggrCmd="irap_single_lib2report_atlas -B -j 4 folders_file=$aux out=$ISL_WORKING_DIR/studies/$studyId/$organism name=$studyId"
+job_name="isl.aggregate.${studyId}.${organism}"
+job_id=$(sbatch --time 7-00:00:00 -J $job_name --mem $slurmMem -c 1 -e ${job_name}.err -o ${job_name}.out --chdir=$ISL_WORKING_DIR -p production --wrap $aggrCmd | awk '{print $4}')
```

Comment:
Wrap command in quotes
```suggestion
job_id=$(sbatch --time 7-00:00:00 -J $job_name --mem $slurmMem -c 1 -e ${job_name}.err -o ${job_name}.out --chdir=$ISL_WORKING_DIR -p production --wrap="$aggrCmd" | awk '{print $4}')
```